tests/test_cmx_3600_adapter.py

```diff
@@ -504,6 +504,23 @@ def test_nucoda_edl_write(self):
 
         self.assertMultiLineEqual(result, expected)
 
+    def test_reels_edl_round_trip_string2mem2string(self):
+        
+        sample_data = r'''TITLE: Reels_Example.01
+
+001  ZZ100_50 V     C        01:00:04:05 01:00:05:12 00:59:53:11 00:59:54:18
+* FROM CLIP NAME:  take_1
+* FROM FILE: S:\path\to\ZZ100_501.take_1.0001.exr
+002  ZZ100_50 V     C        01:00:06:13 01:00:08:15 00:59:54:18 00:59:56:20
+* FROM CLIP NAME:  take_2
+* FROM FILE: S:\path\to\ZZ100_502A.take_2.0101.exr
+'''
+        
+        timeline = otio.adapters.read_from_string(sample_data, adapter_name="cmx_3600")
+        otio_data = otio.adapters.write_to_string(timeline, adapter_name="cmx_3600", style="nucoda")
+        
+        self.assertMultiLineEqual(sample_data,otio_data)
```

Comment:
Does `make lint` catch the missing space after the comma here?
```suggestion
        self.assertMultiLineEqual(sample_data, otio_data)
```

Reply:
doh!